src/components/link/link.tsx

```diff
@@ -29,33 +29,33 @@ export class Link implements InteractiveComponent {
   //
   //--------------------------------------------------------------------------
 
-  /** is the link disabled  */
+  /** When true, interaction is prevented and the component is displayed with lower opacity. */
   @Prop({ reflect: true }) disabled = false;
 
   /**
    * Prompts the user to save the linked URL instead of navigating to it. Can be used with or without a value:
    * Without a value, the browser will suggest a filename/extension
-   * See: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#attr-download
+   * See https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#attr-download.
    */
   @Prop({ reflect: true }) download: string | boolean = false;
 
-  /** optionally pass a href - used to determine if the component should render as a link or an anchor */
+  /** Specifies the URL of the linked resource, which can be absolute or relative. */
```

Comment:
absolute/relative is for paths not URLs right?

Reply:
Correct. Updated the language a bit to ensure its not confused with props.

> Specifies the URL of the linked resource, which can be set as an absolute or relative path.